barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.cpp

```diff
@@ -7,74 +7,6 @@
 
 namespace acir_format {
 
-/**
- * @brief Populate variables and public_inputs in builder given witness and constraint_system
- * @details This method replaces consecutive calls to add_public_vars then read_witness.
- *
- * @tparam Builder
- * @param builder
- * @param witness
- * @param constraint_system
- */
-template <typename Builder>
-void populate_variables_and_public_inputs(Builder& builder,
```

Comment:
Tldr why this can go away?

Reply:
This logic was essentially just moved to the UCB constructor. (Previously we would construct a builder, then call this function to populate some data. Now we just pass this data to the builder constructor. This just makes sense to me but also has to be this way if we ever remove the +1 noir offset).